wiki/Tournaments/RDOC/2016/en.md

```diff
@@ -0,0 +1,359 @@
+---
+tags:
+  - RDOC
+  - RDOC2016
+---
+
+# Royal Danish osu!Cup 2016
+
+![RDOC logo](img/logo.png)
+
+The **Royal Danish osu!Cup 2016** (***RDOC2016***) was a national double-elimination 32-player 1v1 osu!standard tournament hosted by ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273). This was the second installment of the Royal Danish osu!Cup.
```

Comment:
```suggestion
The **Royal Danish osu!Cup 2016** (***RDOC 2016***) was a Danish 1v1 double-elimination osu!standard tournament hosted by ![][flag_DK] [Mayoi Hachikuji](https://osu.ppy.sh/users/2114273). It was the first installment of the Royal Danish osu!Cup.
```

Reply:
changing this to 'first installment' would be wrong, keeping this as second and changing other wording